src/rHelpPanel.ts

```diff
@@ -13,6 +13,7 @@ import * as hljs from 'highlight.js';
 import { config } from './util';
 
 import * as api from './api';
+import { downloadAndUnzipVSCode } from 'vscode-test';
```

Comment:
Here is warned by eslint and downloadAndUnzipVSCode can be removed.

Reply:
thanks, fixed (21182c1f9ab2bda6cfc31c631c34ad466a4647b1)